src/ui/Base/ComponentOptions.ts

```diff
@@ -687,6 +687,11 @@ export class ComponentOptions {
         template = ComponentOptions.loadResultTemplateFromId(id);
       }
     }
+
+    if (template == null) {
+      template = ComponentOptions.loadResultTemplateFromPageTemplatesCache();
```

Comment:
But, say someone use the `contentTemplate` option on a quickview, it's gonna use that code path, right ?

And then potentially try to load a `TemplateList` ?